Essentials/src/config.yml

```diff
@@ -381,6 +381,9 @@ list:
     # Uncomment the line below to simply list all players with no grouping
     #Players: '*'
 
+# Show realnames on /list if player has a nickname
```

Comment:
I think this should still be clarified a bit more. Maybe something like:

"Displays real names in /list next to players who are using a nickname."